src/server.js

```diff
@@ -1,45 +1,44 @@
 import { app } from "./support/setupExpress.js";
 import { query } from "./support/db.js";
-import { sum } from "./sum.js";
-import { setupARouteHandlerDemonstratingValidationWithZod } from "./zodDemo/setupARouteHandlerDemonstratingValidationWithZod.js";
+import pg from "pg";
 
 //You should delete all of these route handlers and replace them according to your own requirements
+const dbURL = process.env.DATABASE_URL;
 
-app.get("/", (req, res) => {
-    res.json({
-        outcome: "success",
-        message: "hello world!  Try /sum/1/2 or /db-check",
-    });
+const client = new pg.Client({
+    connectionString: dbURL,
+    ssl: true,
 });
 
-//just an example route handler.  delete it.
-app.get("/sum/:a/:b", handleGETRequestForSum);
-
-//This jsdoc comment helps vscode figure out the correct types for req and res for autocompletion, etc,
-//when it can't figure it out from context.
-/**
- * @param {import('express').Request} req
- * @param {import('express').Response} res
- */
-function handleGETRequestForSum(req, res) {
-    const answer = sum(parseInt(req.params.a), parseInt(req.params.b));
-    res.json({ answer });
-}
-
-//An example route that makes an SQL query to the db.
-app.get("/db-check", async (req, res) => {
+client.connect();
+
+app.get("/movies/search", async (req, res) => {
     try {
-        const dbResult = await query("select * from my_table");
+        let searchTermValue = req.query.searchTerm;
+        const dbResult = await query(
+            "select * from movies where name like $1 limit 10",
+            [`%${searchTermValue}%`]
+        );
         res.json(dbResult.rows);
     } catch (error) {
-        console.error("error handling db-check: ", error);
-        //don't forget to send a response back to the client!
-        res.status(500).json({ outcome: "error", message: "see server logs" });
+        res.status(500).json({ error: error });
     }
 });
 
-//Delete this, too.  It's just a demo for one way to robustly validate user-submitted data.
-setupARouteHandlerDemonstratingValidationWithZod(app);
+app.post("/movies/:movie_id/comments", async (req, res) => {
+    try {
+        const movieID = parseInt(req.params.movie_id);
+        const commentBody = req.body;
+
+        const dbResult = await query(
+            "INSERT INTO comments (movie_id, author, comment_text) VALUES ($1, $2, $3) RETURNING *",
+            [movieID, commentBody.author, commentBody.comment_text]
```

Comment:
Destructuring the commentBody object to extract the author and comment_text variables could help readability